src/components/HeaderForm.js

```diff
@@ -270,15 +270,10 @@ class HeaderForm extends Component {
       }
     });
     if (value === dataTypes.FILE_UPLOAD) {
-      this.setState((state) => {
-        return {
-          xgFile: state.xgSelect,
-          gbwtFile: state.gbwtSelect,
-          gamFile: state.gamSelect,
-          bedFile: state.bedSelect,
-          dataPath: "upload",
-          dataType: dataTypes.FILE_UPLOAD,
-        };
+      this.setState({
+        ...EMPTY_STATE,
```

Comment:
Won't this clear out any `error` that has been set? Is that likely to be a problem?